File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n = nums.length
    let sum = n*(n+1)/2;
    let a = 0;
    for(let i=0;i<nums.length;i++){
        a =a+nums[i]
        
    }
    return sum - a
};

File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        output = 0
        for i in range(0,len(nums)+1):
            output = output ^ i 
        for i in nums:
            output = output ^ i
        return output  